json_schemas/briefs-digital-outcomes-and-specialists-digital-specialists.json

```diff
@@ -188,6 +194,7 @@
     "numberOfSuppliers",
     "organisation",
     "priceWeighting",
+    "requirementsLength",
```

Comment:
I'd do a migration. That's what we've been doing so far and I think in the long run it helps to keep related code simpler, as it doesn't have to deal with missing fields/different data types and so on.

Keeping in mind the decision we've made about separate migrations I think we can release this first and then do a migration for published briefs. If we do it the other way around there's a risk of a new specialist being published after the migration but before the new field is added.

This also affects unpublished briefs – for example if a brief was ready to publish before now it will requires more information. So it might be worth checking if any comms have to go out beforehand, if you haven't already.